src/js/components/Search.js

```diff
@@ -163,7 +163,10 @@ export default class Search extends Component {
   }
 
   _onInputKeyDown (event) {
```

Comment:
I don't think we should touch the onInputKeyDown.

there is the `_onEnter` callback that I believe is missing an else.

Today the function logic is like this:

```
if (activeSuggestionIndex >= 0) {
      const suggestion = suggestions[activeSuggestionIndex];
      this.setState({ value: suggestion }, () => {
        const suggestionMessage = this._renderLabel(suggestion);
        const selectedMessage = Intl.getMessage(intl, 'Selected');
        announce(`${suggestionMessage} ${selectedMessage}`);
      });
      if (onSelect) {
        onSelect({
          target: this._inputRef || this._controlRef,
          suggestion: suggestion
        }, true);
      }
    }
```

and I believe it should be:

```
if (activeSuggestionIndex >= 0) {
      const suggestion = suggestions[activeSuggestionIndex];
      this.setState({ value: suggestion }, () => {
        const suggestionMessage = this._renderLabel(suggestion);
        const selectedMessage = Intl.getMessage(intl, 'Selected');
        announce(`${suggestionMessage} ${selectedMessage}`);
      });
      if (onSelect) {
        onSelect({
          target: this._inputRef || this._controlRef,
          suggestion: suggestion
        }, true);
      }
    } else {
      onSelect({
        target: this._inputRef || this._controlRef
      }, false);
    }
```